recipes-core/fstools/fstools_git.bb

```diff
@@ -9,17 +9,18 @@ HOMEPAGE = "https://git.openwrt.org/?p=project/fstools.git;a=summary"
 LICENSE = "LGPL-2.1 & GPL-2.0 & PD"
 LIC_FILES_CHKSUM = "file://ubi.c;beginline=1;endline=17;md5=8ccc371d64f0b3a8d91065b678dc7095"
 SECTION = "base"
+DEPENDS += "util-linux"
```

Comment:
this seems to be new if this is needed add a reason to commit

Reply:
oops, will fix that.

Reply:
as noted below it's not actually new, just moved.